index.bs

```diff
@@ -171,7 +190,7 @@ When an item is of type <dfn export for="AV1 Image Item Type">av01</dfn>, it is
 <h4 id="av1-item-configuration-property">AV1 Item Configuration Property</h4>
 
 <pre class="def">
-    Box Type:                 <dfn export for="AV1ItemConfigurationProperty">av1C</dfn>
+    Box Type:                 <dfn export property>av1C</dfn>
```

Comment:
Was this intentional? Won't this cause a conflict with AV1-ISOBMFF which should already be defining `av1C`.

Reply:
This should export a property and AV1-ISOBMFF should export a regular dfn I guess? Otherwise I cannot refer to the `av1C` property from this document.

Reply:
Fixed by using `'[=AV1ItemConfigurationProperty/av1C=]'` instead. Also filed #244 to address that separately and reverted the changes unrelated to this PR.